src/app/search/search-filters/sockets.tsx

```diff
@@ -290,6 +290,20 @@ const socketFilters: FilterDefinition[] = [
       (item: DimItem) =>
         item.sockets && compare!(countEnhancedPerks(item.sockets)),
   },
+  {
+    keywords: 'discontinuedperk',
+    description: tl('Filter.DiscontinuedPerk'),
+    destinyVersion: 2,
+    filter: () => (item: DimItem) => {
+      if (!(item.bucket?.sort === 'Weapons' && item.tier.toLowerCase() === 'legendary')) {
+        return false;
+      }
```

Comment:
I'm not sure about this `toLowerCase()` -- probably should just be `item.tier === 'Legendary'` -- but it seems to have been copied from a different filter so /shrug